scss/6-components/_navigation.scss

```diff
@@ -25,15 +25,15 @@ nav,
         font-weight: $cads-font-weight__bold;
     }
 
-    .nav__dropdown-toggle.priority-nav__dropdown-toggle,
-    .nav__dropdown-toggle.priority-nav__dropdown-toggle:hover {
+    .nav__dropdown-toggle.cads-greedy-nav__dropdown-toggle,
```

Comment:
Should all the `.nav__` classes become `cads-...`. That would keep the naming consistent within the DS.

Reply:
on it.